metpy/constants.py

```diff
@@ -70,11 +72,13 @@
     earth_gravity = g = units.Quantity(1.0, units.gravity).to('m / s^2')
     # Taken from GEMPAK constants
     Re = earth_avg_radius = 6.3712e6 * units.m
+    G = gravitational_constant = units.Quantity(6.674e-11, 'm^3 / kg / s^2')
```

Comment:
Can you replace the hand-coded value here with the value from pint:
```python
G = gravitational_constant = units.newtonian_constant_of_gravitation
```

Reply:
Not sure if I've done this correctly, would be good to check!